### PR TITLE
Add RFC on installation names

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/roadmap/issues/1353

During the onsite we discussed how to improve the installation names concept/process for the next few years.

This RFC should pin down the key decision, being:

**Customers should choose installation names they like from a list we provide.**

### CODE REVIEW

File: installation-names/README.md
Comment (Gacko) [inline installation-names/README.md:23]:
```suggestion
Customers should claim any name they like from our [repository](https://github.com/giantswarm/installation-names). The goal here is to foster adoption of the name.
```

**Review by pipo02mix — COMMENTED** [review]
What about adding a dot explaining that the installation name is something you can not change after the installation is created? so we make sure customers select the name thoughtfully

**Comment by marians** [comment]
> What about adding a dot explaining that the installation name is something you can not change after the installation is created? so we make sure customers select the name thoughtfully

I think that should be customer-facing documentation, which will be provided separately. It's already mentioned in https://github.com/giantswarm/installation-names

File: installation-names/README.md
Comment (calvix) [inline installation-names/README.md:18]: `Should be easy to pronounce, to spell` might be very different across different countries, a word that might sound and spell easily for you might not work for someone else

the current list in the installation-names repo is kinda opinionated and I don't like several names

File: installation-names/README.md
Comment (calvix) [inline installation-names/README.md:33]: who will be doing that PR? is that giant swarm employee or customer?

File: installation-names/README.md
Comment (marians) [reply installation-names/README.md:18]: I tried to make it more precise by adding "in an english language context".

If you can tell _why_ you don't like several names, we can try to derive criteria from that. If you simply don't like some names, please provide a PR to the installation-names repo.

File: installation-names/README.md
Comment (marians) [reply installation-names/README.md:33]: It can be either or.

**Review by whites11 — APPROVED** [review]
Seems legit to me

**Comment by Gacko** [comment]
I actually attended the open session, so I might be biased and therefore abstain from approving.

File: installation-names/README.md
Comment (marians) [reply installation-names/README.md:18]: I'd say that's something we can try to accomplish in the installation-names repo, however it will be hard to avoid completely.

**Comment by marians** [comment]
Waiting for more approval, or questions/comments in case you have any.

**Review by anvddriesch — APPROVED** [review]

**Review by kuosandys — APPROVED** [review]

**Pull request merged**
